static/scss/partials/landing/a.scss

```diff
@@ -201,20 +205,44 @@
             grid-template-columns: 1fr 1fr;
             row-gap: none;
             padding: 0;
-            margin-top: 0;
         }
 
         .c-premium {
-            grid-row-start: 1;
             border: 2px solid $color-purple-40;
+            grid-row-start: 1;
+            margin-top: $layout-sm;
 
             @media #{$mq-md} {
                 grid-row-start: auto;
+                margin-top: 0;
+                justify-self: left;
+            }
+
+            li:nth-child(3) {
+                line-height: 100%;
+            }
+
+            .c-feature-3-subtext {
+                @include text-body-sm;
+                margin: 0;
+                color: $color-grey-40;
+            }
+
+            .c-premium-detail {
+                @include text-body-sm;
+                color: $color-blue-50;
+                font-weight: 700;
+                position: absolute;
+                margin-top: -$layout-lg;
```

Comment:
More of a general tip to keep in mind for the future: if you can, try to use `padding` instead of `margin` to avoid affecting the layout of elements next to the component. That will make it easier to reuse the class in other contexts. (_Especially_ when considering the headache that is [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing).)

I haven't fully internalised it yet myself, but still a good thing to keep in mind. This article is a good read on it: https://non-traditional.dev/encapsulated-css-the-key-to-composable-layouts-94f11c177cc1

Reply:
Yup, I too am a bigger fan of padding > margin, however for this case specifically I set it to `position: absolute` and intentionally shifted it out of the parent module, hence the negative `margin-top` value.

Reply:
Ah gotcha, carry on then :)